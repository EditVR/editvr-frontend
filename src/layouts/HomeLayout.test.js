/**
 * @file HomeLayout.test.js
 * Contains tests for HomeLayout.js.
 */

import React from 'react';
import renderer from 'react-test-renderer';

import { HomeLayout } from './index';

describe('<HomeLayout />', () => {
  it('Matches its snapshot', () => {
    expect(
      renderer
        .create(
          <HomeLayout>
            <div>child</div>
          </HomeLayout>
        )
        .toJSON()
    ).toMatchSnapshot();
  });
});
