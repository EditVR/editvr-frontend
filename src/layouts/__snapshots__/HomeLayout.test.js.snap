// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeLayout /> Matches its snapshot 1`] = `
<div
  className="Home-wrapper-1"
  id="layout__wrapper"
>
  <div
    className="Home-centerColumn-2"
    id="layout__center"
  >
    <header
      className="Header-wrapper-3"
    >
      <img
        alt="EditVR logo"
        className="Header-logo-4"
        src="editvr-logo.svg"
      />
    </header>
    <div>
      child
    </div>
    <footer
      className="Footer-wrapper-5"
    >
      <div
        className="Footer-elements-6"
      >
        <a
          href="https://fourkitchens.com"
        >
          <p
            className="MuiTypography-root-9 MuiTypography-body1-18 Footer-text-8"
            type="body1"
          >
            EditVR is a product of
          </p>
          <img
            alt="Four Kitchens"
            className="Footer-logo-7"
            src="4k-logo-reversed.svg"
          />
        </a>
      </div>
    </footer>
  </div>
</div>
`;
